step-definitions/Hotels/Sprint2SD.js

```diff
@@ -0,0 +1,264 @@
+const { Given, Then, When } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const HotelsHome = require("../../POM/Hotels/HotelsHome");
+
+const hotelsHome = new HotelsHome
+
+// TC30
+
+// Given(/^I am on Hotels landing page$/, async function() {
+//     await browser.url('https://www.hotels.com/');
+//     await browser.pause(3000);   
+// })
+
+When(/^I scroll on Get The App$/, async function() {
+    await hotelsHome.scrollToGetApp();
+    await browser.pause(3000);       
+})
+
+When(/^I enter invalid phone number$/, async function() {
+    await hotelsHome.typePhoneNumber('000000');
+    await browser.pause(3000);       
+})
+
+When(/^I click on Get The App$/, async function() {
+    await hotelsHome.clickGetApp();
+    await browser.pause(3000);       
+})
+
+Then(/^I verify “Please enter a valid phone number.“ error is displayed$/, async function() {
+    expect(await hotelsHome.phoneNumberErrorDisplayed(), 'Valid Number error not displayed').to.be.true;
+})
+
+
+
+
+
+// TC31
+
+When(/^I click on English language$/, async function() {
+    await hotelsHome.clickEnglish();
+    await browser.pause(3000);       
```

Comment:
Avoid using browser.pause(). Its not efficient rather use waitFor() or waitUntil(), these are dynamic.